commands.json

```diff
@@ -0,0 +1,83 @@
+{
+    "$schema": "https://cdn.subsquid.io/schemas/commands.json",
+    "commands": {
+      "clean": {
+        "description": "delete all build artifacts",
+        "cmd": ["npx", "--yes", "rimraf", "lib"]
+      },
+      "generate": {
+        "description": "Generate a squid from an ABI file",
+        "cmd": ["squid-gen-abi"]
+      },
+      "squid-gen-abi": {
+        "description": "Generate a squid from an ABI file",
+        "cmd": ["squid-gen-abi"],
+        "hidden": true
+      },
+      "build": {
+        "description": "Build the squid project",
+        "deps": ["clean"],
+        "cmd": ["tsc"]
+      },
+      "up": {
+        "description": "Start a PG database",
+        "cmd": ["docker-compose", "up", "-d"]
+      },
+      "down": {
+        "description": "Drop a PG database",
+        "cmd": ["docker-compose", "down"]
+      },
+      "migration:apply": {
+        "description": "Apply the DB migrations",
+        "cmd": ["squid-typeorm-migration", "apply"]
+      },
+      "migration:generate": {
+        "description": "Generate a DB migration matching the TypeORM entities",
+        "deps": ["build"],
+        "cmd": ["squid-typeorm-migration", "generate"],
+      },
+      "migration:clean": {
+        "description": "Clean the migrations folder",
+        "cmd": ["npx", "--yes", "rimraf", "./db/migrations"],
+      },
+      "migration": {
+        "deps": ["build"],
+        "cmd": ["squid-typeorm-migration", "generate"],
+        "hidden": true
+      },
+      "codegen": {
+        "description": "Generate TypeORM entities from the schema file",
+        "cmd": ["squid-typeorm-codegen"]
+      },
+      "process": {
```

Comment:
Can I suggest to add a command here to generate the TypeScript facade from the Ink ABI? `squid-ink-typegen` works the same way as `squid-evm-typegen` in the end, we could make it transparent to the user:

```json
      "typegen": {
        "description": "Generate data access classes for an ABI file(s) in the ./abi folder",
        "cmd": ["squid-ink-typegen", "--output", "./src/abi", "--abi", {"glob": "./abi/*.json"}]
      },
```
I think it needs to be tested, I am not sure if Aleksey made changes to it recently.

Reply:
I tentatively checked if it works and it did not. Supplied two (identical) files with different names, got just one TS facade as an output. I'd gladly add that command, but for that the interface of `squid-ink-typegen` must be updated first.